App/Navigation/AppNavigation/Primary/index.ts

```diff
@@ -1,18 +1,42 @@
 import { createStackNavigator } from 'react-navigation'
 
 import TabNavigator from './TabNavigator'
-import Account from './Account'
+
+import Account from '../../../SB/views/UserProfile'
+import NotificationSettings from '../../../SB/views/Notifications'
+import Storage from '../../../SB/views/Storage'
+import Mnemonic from '../../../SB/views/UserProfile/Mnemonic'
+import UpdateAvatar from '../../../SB/views/UserOnBoarding/UpdateAvatar'
+
+import ThreadDetail from '../../../SB/views/ThreadDetail'
+import ThreadsEditFriends from '../../../SB/views/ThreadsEditFriends'
+import Comments from '../../../Containers/Comments'
+import AddCaptionScreen from '../../../Containers/AddCaptionScreen'
+import WalletPicker from '../../../Containers/WalletPicker'
+
+import styles, { headerTintColor } from '../../Styles/NavigationStyles'
 
 const nav = createStackNavigator(
   {
     TabNavigator,
-    Account
+
+    Account,
+    NotificationSettings,
+    Storage,
+    Mnemonic,
+    ChangeAvatar: UpdateAvatar,
+
+    AddFriends: ThreadsEditFriends,
+    ViewThread: ThreadDetail,
+    Comments,
+    ThreadSharePhoto: AddCaptionScreen,
+    WalletPicker
```

Comment:
These screens all exist at this higher level now so we can just push any of them on whenever needed, really for notifications.